das-cli/tests/integration/vault_body_timeout.py

```
@@ -0,0 +1,66 @@
+            "vault_port": port,
+            "service_name": "Vault",
+            "service_command_label": "vault",
+        },
+    )
+    try:
+        manager.get_status()
+    except DockerError as error:
+        if error.__cause__ is not None and not isinstance(error.__cause__, TimeoutError):
+            raise SystemExit(
+                f"DockerError cause was {type(error.__cause__).__name__}"
+            ) from error
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)